exonum/tests/config.rs

```diff
@@ -191,3 +209,24 @@ fn test_generate_full_config_run() {
         panic::resume_unwind(err);
     }
 }
+
+#[test]
+fn test_run_dev() {
+    let artifacts_dir = ".exonum";
+    let db_dir = format!("{}/{}", artifacts_dir, "db");
+    let full_db_dir = full_tmp_folder(&db_dir);
+
+    fs::create_dir_all(Path::new(&full_db_dir)).expect("Expected db temp folder to be created.");
+    let old_db_file = full_tmp_name("1", &db_dir);
+    touch(&old_db_file);
+
+    let result = panic::catch_unwind(|| { run_dev(artifacts_dir); });
```

Comment:
Why do you need `catch_unwind` here?

Reply:
Frankly, I've just copy-pasted that without proper thinking. ) Now I understand it's really needed for cleaning up the temp directory, that contains generated configs for tests. And all the assertions should be wrapped into this `catch_unwind`, so we don't skip teardown process.
So there's something to fix, thanks for noticing